db/dcrpg/pgblockchain.go

```diff
@@ -5829,7 +5829,9 @@ func (pgb *ChainDB) GetExplorerBlock(hash string) *exptypes.BlockInfo {
 		Subsidy:               pgb.BlockSubsidy(b.Height, b.Voters),
 	}
 
-	if pgb.IsDCP0011Active(b.Height) {
+	if data.PoWHash != "" {
```

Comment:
I decided to use this approach over just directly setting the field above to `data.PoWHash` because the release version of dcrd with rpc server version 8.0.0 will not have the field populated in the verbose output.

I believe this approach will allow the faster path when it's backed by a dcrd with rpc server version 8.1.0 without having to explicitly require it.

Reply:
I think that's best too.